### PR TITLE
feat: clean GitHub workflows

### PR DESCRIPTION
**Pull request by bohdanprog**
# Summary
Clean the `.github` folder, and update the pull request template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# No Template
-
-👉 Please follow one of the issue templates provided by the repo - if you are seeing this message, it means you haven't.
-
-If you don't follow the issue template, we may immediately close it.
```
[the remark this answers is not in the document]
Comment (bohdanprog) [reply .github/ISSUE_TEMPLATE.md:5]: Hm, but aren't we using [that](https://github.com/software-mansion/react-native-svg/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml) template for the issue? Or did I miss something?

[the remark this answers is not in the document]
Comment (jakex7) [reply .github/ISSUE_TEMPLATE.md:5]: Yes we are, but GitHub is not forcing that template, user can still access form without template from desktop tools or a direct link https://github.com/software-mansion/react-native-svg/issues/new

[the remark this answers is not in the document]
Comment (bohdanprog) [reply .github/ISSUE_TEMPLATE.md]: Okay, I restored that template, let's leave that in our repo.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -21,8 +21,10 @@ Demonstrate the code is solid. Example: The exact commands you ran and their out
 
 | OS      | Implemented |
 | ------- | :---------: |
-| iOS     |    ✅❌     |
-| Android |    ✅❌     |
+| IOS     |    ✅❌      |
```
[the remark this answers is not in the document]
Comment (bohdanprog) [reply .github/PULL_REQUEST_TEMPLATE.md]: fixed

**Review by jakex7 — APPROVED** [review]
LGTM

**Pull request merged**